src/commands/flex/plugins/start.ts

```diff
@@ -78,6 +81,26 @@ export default class FlexPluginsStart extends FlexPlugin {
     }
   }
 
+  async checkPlugin(pluginName: string) {
+    // read cli plugins json to get directory
+    const config = this.pluginsConfig;
+    const plugin = config.plugins.find((p) => p.name === pluginName);
+
+    if (!plugin) {
+      throw new TwilioCliError(`The plugin ${pluginName} was not found.`);
+    }
+
+    const pkgDir = `${plugin.dir}/package.json`;
+    const pkg = readJSONFile(pkgDir);
+    const scriptVersion = pkg.dependencies['flex-plugin-scripts'];
+
+    if (!semver.satisfies(scriptVersion, '>=4.0.0')) {
```

Comment:
What if `scriptVersion` is empty? (i.e. the customer has no `flex-plugin-scripts`)? Do `if (!scriptVersion || !semver.satisfies(scriptVersion, '>=4.0.0')) {`